app/managers.py

```diff
@@ -0,0 +1,42 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")
+        self.table = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connection.execute(
+            f"INSERT INTO {self.table} (first_name, last_name) "
+            f"VALUES (?, ?) ",
+            (first_name, last_name)
+        )
+        self._connection.commit()
+
+    def all(self) -> list:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table}"
+        )
+        return [
+            Actor(*row) for row in actors_cursor
+        ]
+
+    def update(
+            self, id_to_update: int, new_first_name: str, new_last_name: str
+    ) -> None:
+        self._connection.execute(
+            f"UPDATE {self.table} "
+            "SET first_name = ? AND last_name = ? "
+            "WHERE id = ? ",
+            (new_first_name, new_last_name, id_to_update)
+        )
+        self._connection.commit()
+
+    def delete(self, id_to_delete: int) -> None:
```

Comment:
Also it is not id_to_delete , it is actor id (

Reply:
yes, better when other dev can understand :)